remoting/remoting-triple/src/main/java/com/alipay/sofa/rpc/tracer/sofatracer/TripleTracerAdapter.java

```diff
@@ -274,6 +275,11 @@ public static void serverReceived(SofaRequest sofaRequest, ServerServiceDefiniti
         }
     }
 
+    private static String buildLogServiceName(String interfaceName, String uniqueId) {
```

Comment:
Duplicate code, this function should managed in a certain class.

Reply:
@OrezzerO OK I see, have modified.